gpflow/ci_utils.py

```diff
@@ -18,7 +18,17 @@
 
 
 def is_continuous_integration():
-    return os.environ.get('CI', None) is not None
+    """
+    Returns `True` if notebooks are executed on CI system, otherwise `False`.
+    Also returns `False` is `DOCS` environment variable is set,
+    even when notebooks are executed on CI.
+    """
+    ci = os.environ.get('CI', None)
+    docs = os.environ.get('DOCS', None)
+    if docs is not None:
+        return False
+
+    return (ci is not None)
```

Comment:
could be
```
if 'DOCS' in os.environ:
    return False

return 'CI' in os.environ
```

Reply:
:+1: to the first suggestion (I think the one-liner makes it less clear what's going on)

Reply:
This is neat - thanks!